builtin.c

```diff
@@ -774,6 +774,9 @@ XS(XS_builtin_import)
 void
 Perl_boot_core_builtin(pTHX)
 {
+    HV * inc_hv;
+    GV * ver_gv;
+    SV * ver_sv;
     I32 i;
```

Comment:
Can we fold these declarations into the assignments below now that we're C99? That would seem more readable to me. The loop index should probably also move but that might be considered out of scope for this PR.

Reply:
done in latestest .rev